src/material/tooltip/tooltip.ts

```diff
@@ -222,8 +225,10 @@ export abstract class _MatTooltipBase<T extends _TooltipComponentBase>
     // If tooltip is disabled, hide immediately.
     if (this._disabled) {
       this.hide(0);
+      this._elementRef.nativeElement.classList.add(`${this._cssClassPrefix}-${DISABLED_CLASS}`);
```

Comment:
This should be done through the `host` bindings.

Reply:
Done